### DISCUSSION TITLE
添加Onedrive后出现异常。

### DISCUSSION THREAD

**Discussion by flash168** [AlistGo/alist#769] [Q&A] [answered]
按照教程添加onedrive盘之后查看目录内容，返回错误，错误信息：The resource could not be found.
哪位铁子遇到这个问题，设置问题还是哪的问题？

**Comment by geniusshiun** [comment, marked as answer]
你onedriver的目錄有建立嗎?

**Reply by flash168** [reply]
创建好了，问题是我路径配置错误。谢谢

**Comment by FHYQ-Dong** [comment]
我也出现了这个问题。请问您当时是哪个路径配置错误？谢谢

**Reply by FHYQ-Dong** [reply]
问题解决。
出现此问题需要注意 `账号设置` 中 `根目录路径` 是否有写入权限。
建议直接留空。
